### PR TITLE
Stop copying CRDs from libcalico-go

### PR DESCRIPTION
**Pull request by caseydavenport**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: kube-controllers/Makefile [hunk 1]
```diff
@@ -116,14 +106,14 @@ ut:
...
-fv: remote-deps tests/fv/fv.test image
+fv: tests/fv/fv.test image
 	@echo Running Go FVs.
 	cd tests/fv && ETCD_IMAGE=$(ETCD_IMAGE) \
 		KUBE_IMAGE=$(CALICO_BUILD) \
 		CONTAINER_NAME=$(KUBE_CONTROLLERS_IMAGE):latest-$(ARCH) \
 		MIGRATION_CONTAINER_NAME=$(FLANNEL_MIGRATION_IMAGE):latest-$(ARCH) \
 		PRIVATE_KEY=`pwd`/private.key \
-		CRDS=$(CURDIR)/tests/crds \
+		CRDS=$(CURDIR)/../libcalico-go/tests/crds \
```
Comment (robbrockbank) [inline kube-controllers/Makefile:116]: tests/crds -> config/crd   ?

Comment (caseydavenport) [reply kube-controllers/Makefile:116]: yep!

**Review by robbrockbank — APPROVED** [review]
I think it's fine, but naively I'm more on the side of generating the yaml directly in the docs site, or having a top level manifests folder.

**Review by robbrockbank — CHANGES_REQUESTED** [review]
Looks lie a bunch of tests are failing.

File: apiserver/Makefile [hunk 2]
```diff
@@ -232,7 +227,7 @@ run-kubernetes-server: config/crd run-etcd stop-kubernetes-server
 		-v $(CURDIR)/test/certs:/home/user/certs \
 		lachlanevenson/k8s-kubectl:${KUBECTL_VERSION} \
 		--kubeconfig=/home/user/certs/kubeconfig \
-		apply -f /manifests/config/crd/
+		apply -f /go/src/github.com/projectcalico/calico/libcalico-go/config/crd/
```
Comment (electricjesus) [inline apiserver/Makefile:230]: this doesn't seem to pick up the correct path, seems like `:231` needs to change from the current `-v  $(CURDIR):/manifests`

File: apiserver/Makefile [hunk 3]
```diff
@@ -232,7 +227,7 @@ run-kubernetes-server: config/crd run-etcd stop-kubernetes-server
 		-v $(CURDIR)/test/certs:/home/user/certs \
```
Comment (electricjesus) [inline apiserver/Makefile:227]: I can't add a suggestion directly but it seems like you might need to modify the manifests volume above this line to: 
```suggestion
                -v $(CURDIR)/../:/go/src/github.com/projectcalico/calico:rw
		-v $(CURDIR)/test/certs:/home/user/certs \
```

**Review by electricjesus — COMMENTED** [review]
a few things observed from the CI

**Pull request merged**